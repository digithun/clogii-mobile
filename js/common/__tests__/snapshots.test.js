import React from 'react';
import renderer from 'react-test-renderer';

describe('CircleImageWithCategory', () => {
  const CircleImageWithCategory = require('../CircleImageWithCategory').default;
  it('render CircleImageWithCategory', () => {
    const tree = renderer.create(<CircleImageWithCategory
      source={{uri: 'x.jpg'}}
      categorySource={{uri: 'x.jpg'}}
      size={100}
    />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
