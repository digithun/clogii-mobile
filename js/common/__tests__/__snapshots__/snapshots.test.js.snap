exports[`CircleImageWithCategory render CircleImageWithCategory 1`] = `
<View
  style={
    Object {
      "height": 100,
      "width": 100,
    }
  }>
  <Image
    categorySource={
      Object {
        "uri": "x.jpg",
      }
    }
    size={100}
    source={
      Object {
        "uri": "x.jpg",
      }
    }
    style={
      Array [
        Object {
          "borderRadius": 50,
          "height": 100,
          "width": 100,
        },
        Array [
          undefined,
          Object {
            "backgroundColor": "transparent",
          },
        ],
      ]
    } />
  <Image
    source={
      Object {
        "uri": "x.jpg",
      }
    }
    style={
      Array [
        Object {
          "bottom": 3.125,
          "height": 25,
          "position": "absolute",
          "right": 3.125,
          "width": 25,
        },
        undefined,
      ]
    } />
</View>
`;
